src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewTransferNoteViewModelMap.cs

```diff
@@ -46,6 +46,10 @@ public ViewTransferNoteViewModel Map(ViewTransferNoteViewModelMapTransfer source
                 EvidenceNoteId = source.TransferEvidenceNoteData.Id,
                 SubmittedDate = source.TransferEvidenceNoteData.SubmittedDate.ToDisplayGMTDateTimeString(),
                 ApprovedDate = source.TransferEvidenceNoteData.ApprovedDate.ToDisplayGMTDateTimeString(),
+                RejectedDate = source.TransferEvidenceNoteData.RejectedDate.ToDisplayGMTDateTimeString(),
+                ReturnedDate = source.TransferEvidenceNoteData.ReturnedDate.ToDisplayGMTDateTimeString(), 
+                RejectedReason = source.TransferEvidenceNoteData.RejectedReason, 
+                ReturnedReason = source.TransferEvidenceNoteData.ReturnedReason,
```

Comment:
Will need unit tests around this mapping. Add to TransferEvidenceNoteMapTests, should be examples in EvidenceNoteMapTests

Reply:
@GuidoWorkDev Did you look at the unit tests for these properties?